clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py

```diff
@@ -21,18 +21,14 @@
 PANEL_HEIGHT = g.Pixels(300)
 QUANTILES = [0.99, 0.9, 0.5]
 
+SOURCE_TEMPLATE = g.Template(name="source", type="datasource", query="prometheus")
+
 
 @attr.s
 class Dashboard(g.Dashboard):
     time = attr.ib(default=g.Time("now-30d", "now"))
-    templating = attr.ib(
-        default=g.Templating(
-            list=[
-                # Make it possible to use $source as a source.
-                g.Template(name="source", type="datasource", query="prometheus")
-            ]
-        )
-    )
+    # Make it possible to use $source as a source.
```

Comment:
IIUC you dropped new variables completely.

I didn't mean to remove etcd_operation and etcd_type variables completely -- rather to somehow move them to master-dashboard only.

Reply:
They were moved not deleted, please see bottom of master-dashboard.dashboard.py